motor/providers/azure/client.go

```diff
@@ -18,44 +13,3 @@ func IsAzInstalled() bool {
 	command, err := t.RunCommand("az")
 	return command.ExitStatus == 0 && err == nil
 }
-
-// shells out to `az account show --output json` to determine the default account
-// call `az account list` displays all subscriptions
-func GetAccount() (*AzureAccount, error) {
```

Comment:
Shouldn't that be used by default if users type `cnquery shell azure`?

Reply:
The azure sdk uses `az account get-access-token -o json` to do so, we only need to check if `az` is present

Reply:
and the az command also gather the default subscription?

Reply:
We only use the `IsAzInstalled` to make sure that if you run `cnquery shell azure` without any params you can use the AZ cli. In that case it discovers all subs and you can pick:
![image](https://user-images.githubusercontent.com/11717082/201908515-9ca2ea1d-a168-483d-8e8f-e3e1e7b438f5.png)